### PR TITLE
Test - ruby 3.0/3.1

### PR DESCRIPTION
**Pull request by poorndm**
### Description
A Ruby interface for mutexes on MS Windows.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: .expeditor/verify.pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      docker:
+        image: ruby:2.7
+
+- label: run-specs-ruby-3.0
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:3.0
```
Comment (PrajaktaPurohit) [inline .expeditor/verify.pipeline.yml:44]: we also need to add the tests for 3.1

File: win32-mutex.gemspec [hunk 2]
```diff
@@ -1,6 +1,7 @@
+
 Gem::Specification.new do |spec|
   spec.name       = "win32-mutex"
-  spec.version    = Win32::Mutex::VERSION
+  spec.version    = "0.4.4"
```
Comment (jeremiahsnapp) [inline win32-mutex.gemspec:4]: This shouldn't be changed. We don't want to hardcode the version.

**Review by jeremiahsnapp — CHANGES_REQUESTED** [review]
This looks great! I only have the one request that we don't hardcode the version in the gemspec file.

File: win32-mutex.gemspec (hunk 2 above)
Comment (poorndm) [reply win32-mutex.gemspec]: Reverted the changes and modified version.rb to fix the issue.

**Review by jeremiahsnapp — APPROVED** [review]

**Pull request merged**